sig-security/README.md

```diff
@@ -38,6 +38,12 @@ The Chairs of the SIG run operations and processes governing the SIG.
 ## Subprojects
 
 The following [subprojects][subproject-definition] are owned by sig-security:
+### security-assessments
+Security focussed Assessments of Kubernetes Sub-projects
+- **Owners:**
+  - [kubernetes/sig-security/sig-security-assessments](https://github.com/kubernetes/sig-security/blob/main/sig-security-assessments/OWNERS)
+- **Contact:**
+  - Slack: [#sig-security-assess-capi](https://kubernetes.slack.com/messages/sig-security-assess-capi)
```

Comment:
This feels like it's tying the subproject pretty closely to the history of the CAPI assessment; WDYT about creating a new slack channel for a fresh history and more general focus?.

As we figure out the long-term workflows for self-assessments we can come back and update things as necessary.

Reply:
I would also +1 avoid tying it directly to capi. If the channel has the right folk in it, could just rename it to be more generic.

Reply:
Thanks @tabbysable and @mrbobbytables ! I'll have a look at the slack channel to see its content. If it's general enough, I'll plan to rename. Otherwise I'll make a new one.

If I were to rename the channel, is there a process I need to go through for doing that? Or just rename it and submit a PR to update the README?

Reply:
Just requested a new channel! https://github.com/kubernetes/community/issues/6722